Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/PowderILLParameterScanTest.py

```diff
@@ -94,5 +94,10 @@ def test_rebin(self):
         self.assertEquals(red.getNumberHistograms(), 1)
         self.assertAlmostEqual(red.getAxis(1).extractValues()[0], 248.372, 5)
 
+    def test_normalise_time_single(self):
+        red = PowderILLParameterScan(Run='967087',NormaliseTo='Time')
+        self.assertTrue(red)
+        self.assertAlmostEquals(red.readY(0)[1400],9532/300,4)
```

Comment:
This passes, but doesn't do what it seems, Python 2 evaluates `9532/300` as an int, so the result is 31. This is fixed with Python 3. To make it work for both do `9532/300.0`, which will force the result to be a float in both

Reply:
Great catch!